largestValueTreeLevel.py

```diff
@@ -0,0 +1,46 @@
+# As taught in class I used BFS to find largest node in a binary tree
+# Time Complexity: O(n) as we traverse all nodes level by level
+# Space Complexity: O(n)
+import queue
+class Node:
+    def __init__(self,val):
+        self.val = val
+        self.left = None
+        self.right = None
+class Solution:
+    def findLasrgest(self,root):
+        result = list()
+        if root is None:
+            return
+        bfsq = list()
+        bfsq.append(root)
+        while(len(bfsq)!=0):
+            max = -99999
+            size = len(bfsq)
+
+            for i in range(size):
+                node = bfsq.pop(0)
```

Comment:
Please use a queue or a deque instead, .pop(0) causes shifting and compromises runtime.